plugins/org.eclipse.thym.core/src/org/eclipse/thym/core/HybridProject.java

```diff
@@ -147,6 +147,14 @@ public IFile getConfigFile(){
 		return null;
 	}
 	
+	public IFile getPlatformsJSONFile() {
```

Comment:
After further discussion, it makes sense to me. I've folded the getting platforms.json directly into where it's needed instead.